assets/scss/main.scss

```diff
@@ -170,6 +157,29 @@ a {
   }
 }
 
+.link {
+  @include link;
+}
+
+.link-no-decoration {
+  color: inherit;
+  text-decoration: none;
+
+  &:hover, &:active, &:visited {
+    text-decoration: none;
+  }
+}
+
+.hover-list-item {
```

Comment:
Please name it `widget-item` delete &:focus.
Also, you can use that class in job offers components. Do it, please.